remote.go

```diff
@@ -1278,6 +1278,15 @@ func (wd *remoteWD) SetAlertText(text string) error {
 	return wd.voidCommand("/session/%s/alert/text", data)
 }
 
+func (wd *remoteWD) AddVirtualAuthenticator(params map[string]interface{}) error {
```

Comment:
is it possible adding some tests for this at https://github.com/go-auxiliaries/selenium/blob/master/internal/seleniumtest/seleniumtest.go ?

Reply:
Sure